main/guides/coreeval/permissions.md

```diff
@@ -34,21 +34,21 @@ export const main = startSellConcertTicketsContract;
 
 In the top level promise space, we have:
 
-- **agoricNames**: read-only access to the [agoricNames](../integration/name-services.md#agoricnames-agoricnamesadmin-well-known-names) name service.
+- **agoricNames**: read-only access to the [agoricNames](../integration/name-services#agoricnames-agoricnamesadmin-well-known-names) name service.
 
 - **board**: the [board](../integration/name-services.md#the-board-publishing-under-arbitrary-names) name service.
   **Note: the board only grows; no mechanism to reclaim storage has been established.**
 
-- **chainStorage**: to make storage nodes to write to vstorage.
+- **chainStorage**: to make storage nodes to [write to vstorage](../zoe/pub-to-storage).
   **Warning: this includes the right to over-write previously allocated storage nodes.**
 
-- **chainTimerService**: for getting the current timer and setting timer wake-ups; for example, at the conclusion of a governance vote.
+- **chainTimerService**: for getting the current [timer](../../reference/repl/timerServices) and setting timer wake-ups; for example, at the conclusion of a governance vote.
```

Comment:
Aargh!  That's way out-of-date.  I added a link to the [internal doc](https://github.com/Agoric/agoric-sdk/blob/master/packages/SwingSet/docs/timer.md) to #948.